acstools/satdet.py

```diff
@@ -129,7 +129,7 @@
 #
 from __future__ import (absolute_import, division, print_function,
                         unicode_literals)
-from astropy.extern.six.moves import map
```

Comment:
This is some future-proofing that I am sneaking in here. 😉 Otherwise, it is more work as a new PR, which then we'll have to rebase on top of each other.